db/seeds.d/102-organizations.rb

```diff
@@ -12,12 +12,12 @@
   end
 end
 
-if ENV['SEED_ORGANIZATION']
-  if Setting['db_pending_seed']
-    admin = User.where(:login => ENV['SEED_ADMIN_USER'].present? ? ENV['SEED_ADMIN_USER'] : 'admin').first
-    if admin && admin.default_organization.nil?
-      admin.default_organization = Organization.find_by(:name => ENV['SEED_ORGANIZATION'])
-      admin.save!
-    end
-  end
-end
+#if ENV['SEED_ORGANIZATION']
+#  if Setting['db_pending_seed']
+#    admin = User.where(:login => ENV['SEED_ADMIN_USER'].present? ? ENV['SEED_ADMIN_USER'] : 'admin').first
+#    if admin && admin.default_organization.nil?
+#      admin.default_organization = Organization.find_by(:name => ENV['SEED_ORGANIZATION'])
+#      admin.save!
+#    end
+#  end
+#end
```

Comment:
Whoops not again! :)

Reply:
Oh damn, I need to figure if I still need this workaround :-) sorry, will push the right fix later, probably tomorrow. Quick fixes turns to be time consuming if one does not pay attention.